### DISCUSSION TITLE
部分界面无法用触控笔操作

### DISCUSSION THREAD

**Discussion by HofNature** [xiaoyaocz/flutter_dmzj#136] [General]
在平板电脑使用时，漫画阅读器内的设置界面无法使用触控笔滑动，但可以点击![Screenshot_2023-03-21-02-34-06-346_com.xycz.dmzjx.jpg](https://user-images.githubusercontent.com/35188269/226434640-8a529980-9472-4b68-bf41-1584dd916f2d.jpg)

**Comment by xiaoyaocz** [comment]
没有设备测试，大概是此问题https://github.com/flutter/flutter/issues/42846 ，暂时搁置

**Reply by HofNature** [reply]
其实可以姑且调高这个界面的高度，直接显示全，也就无所谓滑动了。
